editor/src/components/navigator/left-pane.tsx

```diff
@@ -959,14 +1060,36 @@ const GithubPane = React.memo(() => {
               fontSize: '11px',
             }}
           >
-            Work with a Github repo if you have access to it.
+            Connect this project to a Github repository. You can then import and export to the repo
+            if you have the correct permissions. Please note we don’t support connecting to private
+            repositories at the moment.
           </div>
           <UIGridRow padded variant='<-------------1fr------------->'>
-            <StringInput
-              testId='saveToGithubInput'
-              value={targetRepository}
-              disabled={!githubAuthenticated}
+            <CreatableSelect
+              autoFocus={false}
+              placeholder={
+                usersRepositories == null ? 'Loading repositories...' : 'owner/repository'
+              }
+              value={currentTargetRepositoryValue}
```

Comment:
I think this code implies we will show dubious values if the project isn't yet connected to a github repo?

Reply:
The value will be `undefined` in the case where the project isn't yet connected, which results in the placeholder being shown.